src/test/regress/multi_task_tracker_extra_schedule

```diff
@@ -0,0 +1,109 @@
+# ----------
+# $Id$
+#
+# Regression tests that exercise distributed planning/execution functionality.
+#
```

Comment:
Could you add a description about tests in this schedule? Similarly for multi_binary_schedule.